match/tensor.py

```diff
@@ -65,32 +62,32 @@ def _gradient() -> None:
         result._gradient = _gradient
         return result
 
-    def sum(self) -> Matrix:
+    def sum(self) -> Tensor:
         """Return the sum of all values across both dimensions."""
-        result = Matrix(List2D(1, 1, self.data.sum()), children=(self,))
+        result = Tensor(TensorData(value = self.data.sum()), children=(self,))
```

Comment:
Has this file been formatted? This looks different than I would expect from black.